docs/samples/shards/General/CaptureLog/1.edn

```diff
@@ -0,0 +1,28 @@
+(defloop with-ui
+  (GFX.MainWindow
+   :Title "Console demo" :Width 800 :Height 600
+   :Contents
+   (->
+    (Setup
+     (GFX.DrawQueue) >= .ui-draw-queue
+     (GFX.UIPass .ui-draw-queue) >> .render-steps)
+
+    .ui-draw-queue (GFX.ClearQueue)
+
+    (UI
+     .ui-draw-queue
+     (->
+      (UI.CentralPanel
+       (->
+        (UI.Button "Say Hello" (Msg "Hello!"))
+        (CaptureLog
+         :Size 24
```

Comment:
I made `CaptureLog` output a sequence.

For the moment `(UI.Console)` still takes a single string as input. That's because `egui::TextEdit` only works with a single string entry. We could made our own widget and deal with a list of entries instead but that's a lot of work (though we could partially copy-paste `egui::TextEdit` implementation. So for now, I'll keep it simple.